sources/packages/web/src/components/aest/StudentApplications.vue

```diff
@@ -26,7 +28,7 @@
         <v-btn
           v-if="clientType === ClientIdType.Student"
           plain
-          @click="goToApplication(slotProps.data.id)"
+          @click="$emit('goToApplication', slotProps.data.id)"
```

Comment:
I see emit being used multiple times, We can create an emit function and call from all the click.

Reply:
`$emit` is the vue3 in-build function 
![image](https://user-images.githubusercontent.com/77353155/155205794-1c9d5b09-9156-4e1c-b8bf-f99658911f80.png)
.it avoids additions code in setup()

Reply:
Got it. Thanks 👍